data/governance.yml

```
@@ -0,0 +1,522 @@
+    contacts:
+    - name: GitHub
+      link: https://github.com/realworldocaml/mdx
+      kind: github
+    members:
+    - name: Marek Kubica
+      github: Leonidas-from-XIV
+      role: Lead Maintainer
+    - name: Thomas Gazagnaire
+      github: samoht
+      role: Maintainer
+  - id: ocp-indent-dev
+    name: Ocp-indent
+    description: Indentation tool for OCaml, to be used from editors like Emacs and Vim.
+    contacts:
+    - name: GitHub
+      link: https://github.com/OCamlPro/ocp-indent
+      kind: github
+    members:
+    - name: Louis Gesbert 
+      github: AltGr
+      role: Lead Maintainer
+  - id: ocamlfind-dev
+    name: Ocamlfind
+    description: The OCaml findlib library manager 
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml/ocamlfind
+      kind: github
+    members:
+    - name: Gerd Stolpmann
+      github: gerdstolpmann
+      role: Lead Maintainer
+    - name: David Allsopp
+      github: dra27
+      role: Maintainer
+  - id: omp-dev
+    name: Omp
+    description: Convert OCaml parsetrees between different major versions
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml-ppx/ocaml-migrate-parsetree
+      kind: github
+    members:
+    - name: Sonja Heinze
+      github: pitag-ha
+      role: Lead Maintainer
+- id: editors
+  name: Editors
+  description: The Editors team develops IDE extensions for OCaml. OCaml currently has support for Vim, Emacs and VSCode.
+  contacts: []
+  subteams:
+  - id: tuareg-dev
+    name: Tuareg
+    description: Emacs OCaml mode
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml/tuareg
+      kind: github
+    members:
+    - name: Christophe Troestler
+      github: Chris00
+      role: Maintainer
+  - id: vim-ocaml-dev
+    name: vim-ocaml
+    description: Vim runtime files for OCaml
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml/vim-ocaml
+      kind: github
+    members:
+    - name: Rudi Grinberg
+      github: rgrinberg
+      role: Maintainer
+    - name: Fabian Hemmer
+      github: copy
+      role: Maintainer
+- id: packaging
+  name: Packaging
+  description: The Packaging team is responsible to ensure the quality of the OCaml
+    packages ecosystem, and in particular, to review submitions to the Opam repository.
+  contacts:
+  - name: GitHub
+    link: https://github.com/ocaml/opam-repository
+    kind: github
+  members:
+  - name: Anil Madhavapeddy
+    github: avsm
+    role: Lead Maintainer
+  - name: Kate Deplaix
+    github: kit-ty-kate
+    role: Maintainer
+  - name: Marcello Seri
+    github: mseri
+    role: Maintainer
+  - name: Thomas Gazagnaire
+    github: samoht
+    role: Maintainer
+- id: infrastructure
+  name: Infrastructure
+  description: The infrastructure team is responsible to maintain and evolve the infrastructure
+    powering the official OCaml projects, including the servers for OCaml.org and
+    the opam-repository CI.
```

```suggestion
    the `opam-repository` CI.
```
Again, if they can be in monospace. If not, leave it as it is.